src/libraries/System.Text.Json/src/System/Text/Json/ThrowHelper.Serialization.cs

```diff
@@ -296,7 +296,7 @@ public static void ThrowJsonException_JsonRequiredPropertyMissing(JsonTypeInfo p
             Debug.Assert(parent.PropertyCache != null);
 
             // Soft cut-off length - once message becomes longer than that we won't be adding more elements
-            const int CutOffLength = 50;
```

Comment:
Because I've updated the error message so that property names are wrapped with single quotes, I'm also increasing the cutoff so that roughly speaking the same number of properties are being displayed.